global.json

```diff
@@ -1,6 +1,6 @@
 {
     "projects": [ "src" ],
     "sdk": {
-        "version": "1.0.0-preview2-003131"
+        "version": "1.0.3"
```

Comment:
Oh, sorry - I thought we'd set it. Just coincidence that they're the same - I saw the output from the bootstrapped installing the old version. 

It would be good to have a bootstrapped to install the right .NET core SDK, but that's probably a separate issue. 🙂